tests/config.json

```diff
@@ -0,0 +1,33 @@
+{
+  "models": [
+    {
+      "model_name": "TinyLlama/TinyLlama-1.1B-Chat-v1.0",
+      "model_class": "LlamaForCausalLM",
+      "num_key_value_heads": 4,
+      "num_attention_heads": 32,
+      "hidden_size": 2048,
+      "n_layer": 22
```

Comment:
while loading tests, and downloading model based on model card, can we not fetch the rest of the config elements from the json?

Reply:
yes, was thinking about, fixed it. only requirement would be `model_name `and class in the json for new test addition now unless there is new config params.